specs/phase0/beacon-chain.md

```diff
@@ -600,17 +601,17 @@ def bytes_to_uint64(data: bytes) -> uint64:
 
 `def hash_tree_root(object: SSZSerializable) -> Root` is a function for hashing objects into a single root by utilizing a hash tree structure, as defined in the [SSZ spec](../../ssz/simple-serialize.md#merkleization).
 
-#### BLS Signatures
+#### BLS signatures
 
-Eth2 makes use of BLS signatures as specified in the [IETF draft BLS specification draft-irtf-cfrg-bls-signature-04](https://tools.ietf.org/html/draft-irtf-cfrg-bls-signature-04). Specifically, eth2 uses the `BLS_SIG_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_` ciphersuite which implements the following interfaces:
+The [IETF BLS signature draft standard v4](https://tools.ietf.org/html/draft-irtf-cfrg-bls-signature-04) with ciphersuite `BLS_SIG_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_` defines the following functions:
 
-- `def Sign(SK: int, message: Bytes) -> BLSSignature`
-- `def Verify(PK: BLSPubkey, message: Bytes, signature: BLSSignature) -> bool`
+- `def Sign(secret_key: int, message: Bytes) -> BLSSignature`
```

Comment:
Either we use seckey and pubkey or secret_key and public_key but both is inconsistent

Reply:
"pubkey" is a word :) It's often used verbally and in writing. Also the spec has been using it as a first-class word (e.g. see `Validator.pubkey` or the type `BLSPubkey`).

 On the hand I've never heard anyone say "seckey" 😂  A quick Google search confirms the intuition: [115k results for "seckey"](https://www.google.com/search?sxsrf=ALeKk00w_kqogusc-VeMYaAZww3JNcKi_w%3A1607360705611&ei=wWDOX5bfJNyBhbIP_oSpsAo&q=%22seckey%22&oq=%22seckey%22&gs_lcp=CgZwc3ktYWIQAzIGCAAQBxAeMgYIABAHEB4yBggAEAcQHjIGCAAQBxAeMgYIABAHEB4yBggAEAcQHjIGCAAQBxAeMgYIABAHEB4yBggAEAcQHjIGCAAQBxAeOgQIABBHOgIIAFCUEVjpGGDdGmgAcAN4AIABd4gB0QSSAQMyLjSYAQCgAQGqAQdnd3Mtd2l6yAEIwAEB&sclient=psy-ab&ved=0ahUKEwjWw4ifrbztAhXcQEEAHX5CCqYQ4dUDCA0&uact=5) versus [3.2m results for "pubkey"](https://www.google.com/search?ei=LGHOX8_5O_q21fAPo6GEoAU&q=%22pubkey%22&oq=%22pubkey%22&gs_lcp=CgZwc3ktYWIQAzIGCAAQBxAeMgYIABAHEB4yBggAEAcQHjIGCAAQBxAeMgYIABAHEB4yBggAEAcQHjIGCAAQBxAeMgYIABAHEB4yBggAEAcQHjIGCAAQBxAeOgQIABBHOgUIABCRAjoICAAQsQMQkQJQkXZY13xgr39oAHACeAGAAZMBiAGCBZIBAzIuNJgBAKABAaoBB2d3cy13aXrIAQjAAQE&sclient=psy-ab&ved=0ahUKEwjPv6LSrbztAhV6WxUIHaMQAVQQ4dUDCA0&uact=5).

Reply:
We use `privkey` in the validator guide. Not perfect but it matches `pubkey` style. 271k results.

Reply:
Consistency with the validator guide is a good point :) I guess we can consider both `pubkey` and `privkey` to be first-class words (despite their etymological origins as contractions).

@mratsim: Does "privkey" work for you? :)

Reply:
I think that while privkey is used in Eth1 (?), secp256k1 is using seckey extensively https://github.com/bitcoin-core/secp256k1/search?q=seckey for example so anyone who used it is familiar with the term. 
And even in cryptography beyond blockchain, libsodium author is using seckey as CLI parameter for widely distributed and ported tool like minisign https://github.com/jedisct1/minisign/search?q=seckey

So the IETF spec we pushed as a standard, the libraries we sponsored:
- https://tools.ietf.org/html/draft-irtf-cfrg-bls-signature-04#section-2.3
- https://github.com/herumi/bls#keygen
- https://github.com/supranational/blst/blob/41e7e4921bf0d7edccc5e04f45842721dd55c945/bindings/rust/src/lib.rs#L307
- secp256k1 which is used for libp2p are using SecretKey/SK/seckey, and it's even less confusing when shortened.
- And the recent crypto tools for non-cryptographers are using seckey.

Reply:
My suggestion is:

1) Make the spec consistent with the validator guide.
2) Use `secret_key` for both the spec and validator guide. (This is for spec-wide consistency where contractions are avoided.)
3) In a separate PR see if folks are happy with changing `pubkey` to `public_key`. I'm certainly not against but I can see how it is a larger cosmetic change (there are 31 hits for "pubkey" in beacon-chain.md). It is also arguably not a necessary change since "pubkey" is a word. I write and say "pubkey" almost exclusively, not "public key".